nucypher/policy/conditions/ecdsa.py

```diff
@@ -26,6 +36,10 @@ class Schema(ExecutionCall.Schema):
         message = fields.Raw(required=True)
         signature = fields.Str(required=True)
```

Comment:
Should `message` and `signature` basically always be context variables given that we don't know these until request time? The only thing set for the condition at encryption time would be the verifying key and the curve..., is that right?

Reply:
Yes - that's correct.